fury/tests/test_window.py

```diff
@@ -11,6 +13,11 @@
 from fury.decorators import skip_osx, skip_win
 from fury import shaders
 
+from fury.lib import (Renderer, Volume, Actor2D, InteractorEventRecorder,
+                      InteractorStyleImage, InteractorStyleTrackballCamera,
+                      RenderWindow, RenderWindowInteractor, RenderLargeImage,
+                      WindowToImageFilter, Command)
```

Comment:
Not sure what is the goal of all this additional import since they are not used

Reply:
Probably a problem with previous merge. Fixed now